tools/web3-plugin-example/test/unit/contract_method_wrappers.test.ts

```diff
@@ -69,15 +73,14 @@ describe('CustomRpcMethodsPlugin Tests', () => {
 				params: [
 					{
 						input: '0x70a082310000000000000000000000008da5e39ec14b57fb9bcd9aa2b4500e909119795d',
-						data: '0x70a082310000000000000000000000008da5e39ec14b57fb9bcd9aa2b4500e909119795d',
```

Comment:
It [looks like](https://github.com/web3/web3.js/actions/runs/5134850870/jobs/9239561221?pr=6134#step:5:103) the removal of `data` needs to be reverted for this test and `transferAndGetBalances` test

Reply:
This is interesting. I needed this change for the test to pass on my machine! I will investigate more on this. And if you have any clues, please let me know.

Reply:
I'm trying to figure all this out